00_Base/src/services/credentials.service.ts

```diff
@@ -546,15 +662,9 @@ export class CredentialsService {
     versionNumber: VersionNumber,
     credentials: CredentialsDTO,
   ): Promise<ClientVersion> {
-    const existingVersionDetails = clientInformation.clientVersionDetails.find(
-      (v: ClientVersion) => v.version === versionNumber,
-    );
-    if (!existingVersionDetails) {
-      throw new NotFoundError('Version details not found');
-    }
     const clientVersion = await this.getVersionDetails(
       versionNumber,
-      existingVersionDetails.url,
+      credentials.url,
```

Comment:
this method is getting VersionDetails--the credentials.url gets Versions
the VersionDetail url is in the Version object, which is why the previous lookup was happening